Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -182,12 +182,12 @@ class PurchasesOrchestrator {
                           return
                       }
 
-                let paymentDiscount = SKPaymentDiscount(identifier: discountIdentifier,
+                let promotionalOffer = PromotionalOffer(identifier: discountIdentifier,
```

Comment:
What do you think about adding a constructor that takes the `SKPaymentDiscount` instead to simplify this?

Reply:
The `SKPaymentDiscount` isn't actually needed here it all. `PromotionalOffer` will be made into the things later on for SK1 and SK2

Reply:
Oh yeah I misread this.